storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -4,110 +4,153 @@ import { Meta } from "@storybook/blocks";
-Wel wijzigt het aantal kolommen, die daardoor smaller of breder worden.
-Ook kunnen elementen op dat moment van plaats veranderen op de pagina.
+Op deze breakpoints verandert alleen het aantal kolommen.
+Die worden daardoor smaller of breder en de elementen erop dus ook.
+Rond een breakpoint kunnen elementen van plaats veranderen op de pagina.
 
 ### Twee thema’s voor witruimte
 
+Het design system ondersteunt zowel websites als webapplicaties.
+De gewenste layout verschilt nogal tussen beide.
+
 #### Ruimtelijk
 
-Voor websites is het grid behoorlijk ruimtelijk.
-Zo is de huisstijl ontworpen.
+De huisstijl voor online is ontworpen op eenvoud.
+Er zijn weinig opties voor achtergrondkleur of belijning.
+Typografie en witruimte blijven over als middelen om schermen overzichtelijk te maken.
+Voor websites is het grid behoorlijk daarom ruimtelijk.
 
-De horizontale witruimte tussen kolommen is 16 pixels breed bij een vensterbreedte van 320 pixels.
+De horizontale witruimte tussen kolommen is 16 pixels breed bij een vensterbreedte van 320 pixels of minder.
 Voor elke 256 pixels extra breedte groeit de witruimte lineair met 8 pixels.
+(Dit komt neer op 3 en 3/8 procent van de schermbreedte plus 6 pixels.)
 Bij een vensterbreedte van 1600 pixels is het dus 56 pixels.
-In vensters die breder zijn dan dat groeit de witruimte niet verder.
+Daarna groeit de witruimte niet verder.
 
 In brede vensters worden de witruimtes bijna even breed als de kolommen zelf.
-Dat lijkt misschien onnatuurlijk, maar is geen probleem – doorgaans worden elementen op 3 of 4 kolommen van het grid geplaatst en die krijgen dan ook de tussenliggende witruimtes mee.
+Dat lijkt misschien onnatuurlijk, maar is geen probleem.
+Elementen bezetten vaak meerdere kolommen van het grid en krijgen dan ook de tussenliggende witruimtes mee.
 
 #### Compact
 
 Voor applicaties is zo veel witruimte niet nodig, zelfs contraproductief.
-Daarom is er een compact thema voor het grid.
+Daarom is er ook een compact thema voor het grid.
 
-Hier begint de horizontale witruimte op 4 pixels bij een vensterbreedte van 320 pixels,
-en groeit deze met 4 pixels per 256 pixels extra breedte.
-Het maximum is hier 40 pixels en dat wordt pas bereikt bij een vensterbreedte van 2.624 pixels.
+De minimale witruimte tussen kolommen is ook hier 16 pixels.
+Maar deze begint pas op een vensterbreedte van 1088 pixels te groeien.
+De toename gaat ook langzamer: 4 pixels per 256 pixels extra breedte.
+(Dit komt neer op 1 en 9/16 procent van de schermbreedte min 1 pixel.)
+Het maximum is hier 40 pixels en dat wordt pas bereikt bij een vensterbreedte van 2624 pixels.
 
-Alle andere eigenschappen van de compacte variatie zijn gelijk aan de ruimtelijke.
+Alle andere eigenschappen van het compacte thema zijn gelijk aan die van het ruimtelijke.
 
 ### Marges links en rechts
 
-#### Voor websites
-
-Het grid centreert zichzelf horizontaal.
+Het grid staat horizontaal gecentreerd.
```

Comment:
Also for the compact theme? Judging from the design I was thinking of left aligned.

Reply:
It’s subtle. The grid is always centred (through `align-self`), but if there is no maximum width, the effect is invisible.

Teams are free to choose the spacious or compact theme, and free to constrain the max width of the grid or not.

I reformulated this part.